### ISSUE TITLE
Web-Files in different path then Jellyfin expects on Fedora

### ISSUE THREAD

**Issue by mihawk90** [jellyfin/jellyfin-web#2059]
**Describe The Bug**
The Jellyfin-Web RPM appears to put the web files into a different location then the Jellyfin-Server RPM expects. Both are downloads of the 10.6.4 RPMs.
Jellyfin-Web puts the data into `/usr/share/jellyfin-web`, but the server seems to expect them in `/usr/lib64/jellyfin/jellyfin-web`.

**Steps To Reproduce**
1. Download and Install jellyfin-server.rpm
2. Download and Install jellyfin-web.rpm
3. Open Terminal and execute `jellyfin`
4. Server does not start with a Fatal Error that the web content directory is invalid or empty (see below).
5. Execute `jellyfin --webdir=/usr/share/jellyfin-web` and start the server with that

**Expected Behavior**
The server should start with the default configuration

**Logs**
```log
[09:12:14] [INF] [1] Main: Jellyfin version: 10.6.4
[09:12:14] [INF] [1] Main: Environment Variables: ["[JELLYFIN_LOG_DIR, /home/tarulia/.local/share/jellyfin/log]"]
[09:12:14] [INF] [1] Main: Arguments: ["/usr/lib64/jellyfin/jellyfin.dll"]
[09:12:14] [INF] [1] Main: Operating system: Linux
[09:12:14] [INF] [1] Main: Architecture: X64
[09:12:14] [INF] [1] Main: 64-Bit Process: True
[09:12:14] [INF] [1] Main: User Interactive: True
[09:12:14] [INF] [1] Main: Processor count: 16
[09:12:14] [INF] [1] Main: Program data path: /home/tarulia/.local/share/jellyfin
[09:12:14] [INF] [1] Main: Web resources path: /usr/lib64/jellyfin/jellyfin-web
[09:12:14] [INF] [1] Main: Application directory: /usr/lib64/jellyfin/
[09:12:14] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /home/tarulia/.cache/jellyfin
[09:12:14] [FTL] [1] Main: Error while starting server.
System.InvalidOperationException: The server is expected to host the web client, but the provided content directory is either invalid or empty: /usr/lib64/jellyfin/jellyfin-web. If you do not want to host the web client with the server, you may set the '--nowebclient' command line flag, or set'hostwebclient=false' in your config settings.
   at Jellyfin.Server.Program.StartApp(StartupOptions options)
[09:12:14] [INF] [1] Emby.Server.Implementations.ApplicationHost: Disposing CoreAppHost
[09:12:14] [INF] [2] Main: Received a SIGTERM signal, shutting down
```

**System (please complete the following information):**
 - Platform: Fedora (32 and 33)
 - Browser: Vivaldi/Chromium
 - Jellyfin Version: 10.6.4

**Additional Context**
This installation was an update from an older version (10.4.something I think?), so it might be a leftover config file overwriting the path. In that case however I can neither find the responsible config file, nor should this be the case during an update and there should be a transition (or a note in the documentation or Release-Notes, which I could not find either).

Sidenote: Is the `hostwebclient` config setting in the error message still correct? The system.xml has an `AutoRunWebApp` config, which sounds to be the same?

**Comment by mihawk90** [comment]
Still an issue since there was no new release (if there has been a fix on master I cannot currently test it).

**Comment by mihawk90** [comment]
Just upgraded to 10.7.1 and the issue remains

**Comment by mihawk90** [comment]
Issue unchanged:

```
[tarulia@localhost ~]$ jellyfin
[14:35:27] [INF] [1] Main: Jellyfin version: 10.7.6
[14:35:27] [INF] [1] Main: Environment Variables: ["[JELLYFIN_LOG_DIR, /home/tarulia/.local/share/jellyfin/log]"]
[14:35:27] [INF] [1] Main: Arguments: ["/usr/lib64/jellyfin/jellyfin.dll"]
[14:35:27] [INF] [1] Main: Operating system: Linux
[...]
[14:35:27] [INF] [1] Main: Web resources path: /usr/lib64/jellyfin/jellyfin-web
[14:35:27] [INF] [1] Main: Application directory: /usr/lib64/jellyfin/
[...]
[14:35:27] [FTL] [1] Main: Error while starting server.
System.InvalidOperationException: The server is expected to host the web client, but the provided content directory is either invalid or empty: /usr/lib64/jellyfin/jellyfin-web. If you do not want to host the web client with the server, you may set the '--nowebclient' command line flag, or set'hostwebclient=false' in your config settings.
   at Jellyfin.Server.Program.StartApp(StartupOptions options)
[14:35:27] [INF] [1] Emby.Server.Implementations.ApplicationHost: Disposing CoreAppHost
[14:35:27] [INF] [2] Main: Received a SIGTERM signal, shutting down
```

**Comment by mihawk90** [comment]
Unchanged:

```
[tarulia@localhost ~]$ jellyfin
[08:45:34] [INF] [1] Main: Jellyfin version: 10.7.7
[08:45:34] [INF] [1] Main: Environment Variables: ["[JELLYFIN_LOG_DIR, /home/tarulia/.local/share/jellyfin/log]"]
[08:45:34] [INF] [1] Main: Arguments: ["/usr/lib64/jellyfin/jellyfin.dll"]
[08:45:34] [INF] [1] Main: Operating system: Linux
[08:45:34] [INF] [1] Main: Architecture: X64
[08:45:34] [INF] [1] Main: 64-Bit Process: True
[08:45:34] [INF] [1] Main: User Interactive: True
[08:45:34] [INF] [1] Main: Processor count: 16
[08:45:34] [INF] [1] Main: Program data path: /home/tarulia/.local/share/jellyfin
[08:45:34] [INF] [1] Main: Web resources path: /usr/lib64/jellyfin/jellyfin-web
[08:45:34] [INF] [1] Main: Application directory: /usr/lib64/jellyfin/
[08:45:35] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /home/tarulia/.cache/jellyfin
[08:45:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN addresses : [127.0.0.1/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]
[08:45:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN exclusions : []
[08:45:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using LAN addresses: [127.0.0.1/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]
[08:45:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind addresses: []
[08:45:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind exclusions: []
[08:45:35] [FTL] [1] Main: Error while starting server.
System.InvalidOperationException: The server is expected to host the web client, but the provided content directory is either invalid or empty: /usr/lib64/jellyfin/jellyfin-web. If you do not want to host the web client with the server, you may set the '--nowebclient' command line flag, or set'hostwebclient=false' in your config settings.
   at Jellyfin.Server.Program.StartApp(StartupOptions options)
[08:45:35] [INF] [1] Emby.Server.Implementations.ApplicationHost: Disposing CoreAppHost
[08:45:35] [INF] [2] Main: Received a SIGTERM signal, shutting down
```

**Comment by mihawk90** [comment]
OK so while I'm working on #3571 I thought... why not fix this along the way. But then it crossed my mind "why is noone else complaining about this?", so I looked into it.

Turns out that when running jellyfin as a service, this works perfectly fine because the package defines some environment variables:
https://github.com/jellyfin/jellyfin/blob/3fc8652c265fe0b9c6876f0b050aed9ef31b187c/fedora/jellyfin.env#L23-L24

And moves them in the right spot:
https://github.com/jellyfin/jellyfin/blob/3fc8652c265fe0b9c6876f0b050aed9ef31b187c/fedora/jellyfin.spec#L85
https://github.com/jellyfin/jellyfin/blob/3fc8652c265fe0b9c6876f0b050aed9ef31b187c/fedora/jellyfin.spec#L102

But evidently these are ignored when running `jellyfin` as a user like I did.

Question now is... is this intended?
